salt/orch/kubernetes.sls

```diff
@@ -12,10 +12,11 @@
 {%- set additional_etcd_members = salt.caasp_etcd.get_additional_etcd_members(num_wanted=num_etcd_members,
                                                                               etcd_members=etcd_members) %}
 
-# Ensure the node is marked as bootstrapping
+# Ensure all the nodes are marked with a 'bootstrap_in_progress' flag
 set-bootstrap-in-progress-flag:
   salt.function:
-    - tgt: '*'
+    - tgt: 'roles:(ca|admin|kube-master|kube-minion|etcd)'
```

Comment:
I'm not sure if we should include the `ca` in this set... It won't harm though I guess.

Reply:
Yeah, I thought the same...